test/test_mkldnn.py

```diff
@@ -578,6 +578,28 @@ def test_transpose(self):
                     x.to_mkldnn().transpose(dim1, dim2).to_dense(),
                 )
 
+    def test_linear_non_contiguous_weight(self):
+        in_features = torch.randint(3, 10, (1,)).item()
+        out_features = torch.randint(3, 100, (1,)).item()
+        x = torch.randn(3, in_features, dtype=torch.float32) * 10
+        w = torch.randn(in_features, out_features, dtype=torch.float32)
+        # Only works for bias case by dispatch onednn linear to addmm.
```

Comment:
This comment can be removed which aten::linear has been ported, and also can add **bias = False** case